index.html

```diff
@@ -87,6 +87,14 @@
       <div class="medal">
         <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
       </div>
+      <ul class="supporters">
```

Comment:
Whoops - sorry for being unclear! 

I'd like to take the whole 'supporters' section you're submitting and place it above the medal class. Consider that as our footer.

The order of the page should be:
* How can I help?
* Interested in learning to code?
* Supporters (your entire section)
* Support
* OC Medal (aka footer)